src/screens/WalletConnectApprovalSheet.js

```diff
@@ -270,13 +270,8 @@ export default function WalletConnectApprovalSheet() {
   const handleConnect = useCallback(() => {
     handled.current = true;
     goBack();
-    if (IS_IOS) {
```

Comment:
moved this inside wc approval callback. i noticed that this was causing the "connection successful" modal to pop up even if there was an error